ARCs/arc-0012.md

```diff
@@ -0,0 +1,200 @@
+---
+arc: 12
+title: Standard for Claimable Algorand Standard Assets (ASA)
+description: Conventions for a TEAL smart signature contract account to receive and disburse claimable Algorand Smart Assets (ASA) to an intended receipient Algorand account.
+authors: Brian Whippo (@silentrhetoric), Joe Polny (@joe-p)
+discussions-to: N/A
+status: Draft
+type: Standards Track
+category: ARC
+created: 2022-09-05
+---
+
+# Claimable Algorand Standard Assets (ASAs)
+
+## Summary
+
+A standard for TEAL contract accounts to serve as a pending claimable ASAs account for Algorand Standard Assets for an intended receiver that is not opted in to the ASA.
+
+## Abstract
+
+The goal of this standard is to establish a standard in the Algorand ecosytem by which ASAs can be sent to an intended receiver even if their account is not opted in to the ASA.  
+
+TEAL stateless smart signature contract accounts corresponding to any Algorand account serve as a claimable ASAs account with logic that permits opting into incoming ASAs and subsequently for only the intended receiver to claim the ASA into their normal account.  A single associated stateful application on the network enables a cost--the minimum balance requirement to create an account and hold an ASA--to be refunded to the ASA creator when the asset is claimed by the receiver.
+
+If integrated into ecosystem technologies including wallets, epxlorers, and dApps, this standard can provide enhanced capabilities around ASAs which are otherwise strictly bound at the protocol level to require opting in to be received.  This also enables the ability to "burn" ASAs by sending them to the claimable ASAs account associated with the global Zero Address.
+
+## Motivation
+
+Algorand requires users to opt in to receive any ASA, a fact which simultaneously:
+
+1. Grants account holders fine-grained control over the contents of their account by allowing them to select what assets to allow and preventing any unwanted tokens from appearing in their balance.
+2. Annoys users when transactions fail and must be resubmitted after completing an opt in transaction--or asking someone else to do so.  This routine is jarring to people who are new to Algorand or are accutomed other blockchains without this requirement.
```

Comment:
```suggestion
2. Frustrates users and developers when accounting for this requirement especially since other blockchains do not have this requirement.
```